test/lib/DeployUtil.test.ts

```diff
@@ -254,14 +254,14 @@ describe('DeployUtil', () => {
 
   it('Should not allow for to deserialize a deploy from JSON with a wrong deploy hash', function () {
     let deploy = testDeploy();
-    let json = DeployUtil.deployToJson(deploy);
+    let json = JSON.parse(JSON.stringify(DeployUtil.deployToJson(deploy)));
```

Comment:
this sucks, but this is the easiest way to do full serialization / deserialization of JSON in this test.